modules/dnn/perf/perf_net.cpp

```diff
@@ -121,7 +121,9 @@ PERF_TEST_P_(DNNTestNetwork, Inception_5h)
 
 PERF_TEST_P_(DNNTestNetwork, ENet)
 {
-    if (backend == DNN_BACKEND_INFERENCE_ENGINE) throw SkipTestException("");
+    if (backend == DNN_BACKEND_INFERENCE_ENGINE ||
+        backend == DNN_BACKEND_DEFAULT && target == DNN_TARGET_OPENCL_FP16)
```

Comment:
Please use more brackets to make static code analyzers happy.

Reply:
fixed